python/dask_cudf/dask_cudf/tests/test_groupby.py

```diff
@@ -321,6 +321,6 @@ def test_groupby_reset_index_drop_True():
     pddf = dd.from_pandas(df.to_pandas(), 5)
     gr = ddf.groupby(["a"]).agg({"b": ["count"]}).reset_index(drop=True)
     pr = pddf.groupby(["a"]).agg({"b": ["count"]}).reset_index(drop=True)
-    gf = gr.compute()
-    pf = pr.compute()
+    gf = gr.compute().sort_values(by=["b"]).reset_index(drop=True)
```

Comment:
Are we dropping the index because we don't currently retain the index on groupbys for cuDF?

Reply:
No, this is for index drop test and `sort_values(by=["b"])` will add an index/column as far as I remember

Reply:
Oh woops, completely missed the test name, thanks!